.github/workflows/deploy.yml

```
@@ -0,0 +1,67 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Unit Test
+      - UI Test
+    branches:
+      - master
+      - development
```

I think it should be `main` and `develop` as you set them at the moment.

```suggestion
    branches:
      - main
      - develop
```

Or they can be detected automatically with those keywords 🤔

---

Fixed in 7e224f0